src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -32,6 +32,8 @@ export interface Props {
   willLoadMoreResults?: boolean;
   /** Is rendered when there are no options */
   emptyState?: React.ReactNode;
+  /** Whether the attached text field should be disabled */
```

Comment:
This doesn't actually disable the text field, does it? It just prevents the popover form showing.

Reply:
Correct, since the text field is passed in as a prop you would have to disable it separately.